example/android/build.gradle

```diff
@@ -0,0 +1,29 @@
+buildscript {
+    ext.kotlin_version = '1.3.50'
+    repositories {
+        google()
+        jcenter()
```

Comment:
it is deprecated. https://stackoverflow.com/a/67482465

Reply:
Ok strange. I just freshly created the project using flutter 2.2.2.
Will replace it